Workbooks/Azure Web Application Firewall/WAF monitoring/WAF Monitoring.workbook

```diff
@@ -0,0 +1,831 @@
+            "queryType": 1,
+            "resourceType": "microsoft.resourcegraph/resources"
+          },
+          {
+            "id": "a125fc08-be6d-4b8b-87e2-7e0cd957db47",
+            "version": "KqlParameterItem/1.0",
+            "name": "DefaultWorkspace_Internal",
+            "type": 1,
+            "isRequired": true,
+            "query": "where type =~ 'microsoft.operationalinsights/workspaces'\r\n|take 1\r\n|project id",
+            "crossComponentResources": [
+              "{Subscription}"
+            ],
+            "isHiddenWhenLocked": true,
+            "timeContextFromParameter": "TimeRange",
+            "queryType": 1,
+            "resourceType": "microsoft.resourcegraph/resources"
+          },
+          {
+            "id": "65674a40-2869-4867-a24d-f86f05fd0354",
+            "version": "KqlParameterItem/1.0",
+            "name": "Workspaces",
+            "type": 5,
+            "isRequired": true,
+            "multiSelect": true,
+            "quote": "'",
+            "delimiter": ",",
+            "query": "where type =~ 'microsoft.operationalinsights/workspaces'\r\n| project id, selected = iff(id =~ '{DefaultWorkspace_Internal}', true, false)\r\n",
+            "crossComponentResources": [
+              "value::selected"
+            ],
+            "typeSettings": {
+              "additionalResourceOptions": [
+                "value::all"
+              ],
+              "showDefault": false
+            },
+            "timeContextFromParameter": "TimeRange",
+            "queryType": 1,
+            "resourceType": "microsoft.resourcegraph/resources",
+            "value": [
+              ""
+            ]
+          },
+          {
+            "id": "49e2f511-592f-4d7f-8fda-d686803f3dbf",
+            "version": "KqlParameterItem/1.0",
+            "name": "TimeRange",
+            "label": "Time Range",
+            "type": 4,
+            "isRequired": true,
+            "value": {
+              "durationMs": 5707080000,
+              "endTime": "2023-01-06T18:50:00.000Z"
+            },
+            "typeSettings": {
+              "selectableValues": [
+                {
+                  "durationMs": 300000
+                },
+                {
+                  "durationMs": 900000
+                },
+                {
+                  "durationMs": 1800000
+                },
+                {
+                  "durationMs": 3600000
+                },
+                {
+                  "durationMs": 14400000
+                },
+                {
+                  "durationMs": 43200000
+                },
+                {
+                  "durationMs": 86400000
+                }
+              ],
+              "allowCustom": true
+            }
+          },
+          {
+            "id": "604a42a0-deca-4a95-a15f-8977646a7fac",
+            "version": "KqlParameterItem/1.0",
+            "name": "WAFType",
+            "type": 5,
```

Comment:
this parameter is set to be a "resource picker" but does not return azure resources, it returns strings.  it even will show a warning like this inthe editor:
![image](https://user-images.githubusercontent.com/10158007/214455572-0d4d9e41-a73b-4551-ad45-cbd8f6f23c92.png)
i think you want the parameter type here to just be "dropdown"